src/components/Root/FXHR.test.js

```diff
@@ -47,4 +55,29 @@ describe('FXHR', () => {
     expect(openSpy.mock.calls).toHaveLength(1);
     expect(aelSpy.mock.calls).toHaveLength(1);
   });
+
+  it('Does not rotate if token is valid', () => {
+    getTokenExpiry.mockResolvedValue({
+      atExpires: Date.now() + (10 * 60 * 1000),
+      rtExpires: Date.now() + (10 * 60 * 1000),
+    });
+
+    testXHR.addEventListener('abort', mockHandler);
+    testXHR.open('POST', 'okapiUrl');
+    testXHR.send(new ArrayBuffer(8));
+    expect(openSpy.mock.calls).toHaveLength(1);
+    expect(aelSpy.mock.calls).toHaveLength(1);
+    expect(rtr.mock.calls).toHaveLength(0);
+  });
+
+  it('Handles Token Errors', () => {
+    rtr.mockImplementationOnce(() => { throw new RTRError('test rotation failure'); });
+    try {
+      testXHR.addEventListener('abort', mockHandler);
+      testXHR.open('POST', 'okapiUrl');
+      testXHR.send(new ArrayBuffer(8));
+    } catch (err) {
+      expect(err instanceof RTRError).toBeTrue;
+    }
```

Comment:
Even if the `catch` clause never executes, this test will pass because it won't contain any failing expectations. Maybe set it up like 
```
const err = null;
try { ... }
catch(e => { err = e; }
finally { expect(err) }
```
Also `s/toBeTrue/toBeTruthy()/` or `toBe(true)`; `toBeTrue` is not a function and is not invoked as a function.